### PR TITLE
mapItems added to methods

### PR DESCRIPTION
**Pull request by PayamOWOW**
I also fixed `@` to relative path problem with `tsc-alias`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
         "rollup-plugin-copy": "^3.4.0",
         "rollup-plugin-typescript2": "^0.31.1",
         "ts-jest": "^27.0.7",
+        "tsc-alias": "^1.4.2",
```
Comment (thomasowow) [inline package.json:39]: @barisbora you know if there is a rollup plugin for this?

**Review by thomasowow — COMMENTED** [review]
Could you check if we could incorporate this alias thing in rollup?

File: package.json (hunk 1 above)
Comment (barisbora) [reply package.json:39]: I saw somewhere, I'll check

**Comment by PayamOWOW** [comment]
@thomasowow I tried many plugins unfortunately non of them worked:
`@rollup/plugin-alias` , `rollup-plugin-import-alias` , `rollup-plugin-paths` , `rollup-plugin-typescript-paths` , `rollup-plugin-typescript`

except that `tsc-alias`

**Comment by thomasowow** [comment]
Fixed in 750e5214b03df411b002b3a33ee5594a247040eb

**Pull request closed without merging**